### PR TITLE
Add feature toggling using flags in DB

### PR DESCRIPTION
**Pull request by mscharfstein**
Similar to how NYS does toggling right now (they mostly do so using user group permissioning from the DB).

Works on #8

### CODE REVIEW

File: src/web/src/locales/es/translation.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
   "Hello World": "Hola mundo",
+  "How are you": "¿Cómo estás?",
```
[the remark this answers is not in the document]
Comment (mscharfstein) [reply src/web/src/locales/es/translation.json:3]: lol

File: src/web/src/components/Item.jsx [hunk 2]
```diff
@@ -43,6 +45,16 @@ function Item(props) {
       });
   }, [id]);
 
+  useEffect(() => {
+    axios.get('/api/v1/flags/search/findByName?name=' + howAreYouFlag)
```
[the remark this answers is not in the document]
Comment (mscharfstein) [reply src/web/src/components/Item.jsx:49]: do you mean the useEffect code? pass a URL and run axios.get with that URL? we could definitely extract that in the future

File: src/web/src/components/Item.jsx [hunk 3]
```diff
@@ -7,8 +7,10 @@ import i18n from '../i18n';
 import { Dropdown } from 'semantic-ui-react'
 
 function Item(props) {
+  const howAreYouFlag = "show_how_are_you";
```
[the remark this answers is not in the document]
Comment (mscharfstein) [reply src/web/src/components/Item.jsx:10]: good idea! i added a TODO and will look into this; will send in a separate PR

File: src/web/src/components/Item.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (kebrahim) [reply src/web/src/components/Item.jsx:50]: i suppose i meant the whole block, if possible (lines 48-56). i can just foresee wanting to call this method in lots of places (any time we want to wrap something in a flag), so we'll want this to be really easy to call

**Pull request merged**